fe/src/main/java/org/apache/doris/alter/SchemaChangeHandler.java

```diff
@@ -841,12 +842,16 @@ private void createJob(long dbId, OlapTable olapTable, Map<Long, LinkedList<Colu
         
         // property 3: timeout
         long timeoutSecond = PropertyAnalyzer.analyzeTimeout(propertyMap, Config.alter_table_timeout_second);
+        boolean changeStorageFormat = PropertyAnalyzer.analyzeStorageFormat(propertyMap);
 
         // create job
         Catalog catalog = Catalog.getCurrentCatalog();
         long jobId = catalog.getNextId();
         SchemaChangeJobV2 schemaChangeJob = new SchemaChangeJobV2(jobId, dbId, olapTable.getId(), olapTable.getName(), timeoutSecond * 1000);
         schemaChangeJob.setBloomFilterInfo(hasBfChange, bfColumns, bfFpp);
+        if (changeStorageFormat) {
```

Comment:
Add some comment for the reason of this code, because it would be difficult to understand in the future.

Reply:
ok